bee-api/bee-rest-api/Cargo.toml

```diff
@@ -27,24 +27,29 @@ bee-storage = { version = "0.12.0", path = "../../bee-storage/bee-storage", defa
 bee-tangle = { version = "0.3.0", path = "../../bee-tangle", default-features = false, optional = true }
 
 async-trait = { version = "0.1.51", default-features = false, optional = true }
+auth-helper = { version = "0.2.0", default-features = false }
+axum = { version = "0.4.8", default-features = false, features = [ "json", "headers" ], optional = true }
```

Comment:
Can you update to latest please ? Better do the changes now if needed

Reply:
updated